examples/step-87/doc/intro.dox

```diff
@@ -0,0 +1,268 @@
+
+The relevant class during the local evaluation is
+Utilities::MPI::RemotePointEvaluation::CellData. It allows to loop over
+cells that surround the points. On these cells,
+a cell iterator and the positions in the reference cell of the
+request points can be queried. Furthermore, this class provides
+controlled access to the output vector of the
+Utilities::MPI::RemotePointEvaluation::evaluate_and_process() function.
+@code
+for (const auto cell_index : cell_data.cell_indices())
+  {
+    const auto cell         = cell_data.get_active_cell_iterator(cell_index);
+    const auto unit_points  = cell_data.get_unit_points(cell_index);
+    const auto local_output = cell_data.get_data_view(cell_index, output);
+  }
+@endcode
+
+The functions
+@code
+const auto evaluated_values =
+  VectorTools::point_values<n_components>(rpe, dof_handler, vector);
+
+const auto evaluated_gradients =
+  VectorTools::point_gradients<n_components>(rpe, dof_handler, vector);
+@endcode
+
+evaluate the values and gradients of a solution defined by DoFHandler and a
+vector at the request points. Internally, a lambda function is passed to
+Utilities::MPI::RemotePointEvaluation.
+Additionally it handles the special case, if points belong to multiple cells
+by taking, e.g., the average, the minimum, or
+the maximum via an optional argument of type EvaluationFlags::EvaluationFlags.
+
+<h3>Motivation: two-phase flow</h3>
+
+The exemplary mini codes presented in this tutorial
+are motivated by the application of two-phase-flow simulations formulated in
+a one-fluid setting using an Eulerian framework. In diffuse interface methods,
+the two phases may be implicitly described by a level-set function, here
+chosen as a signed distance function $\phi(\boldsymbol{ x})$ in
+$\Omega$ and illustrated for a popular benchmark case of a rising
+bubble in the following figure.
+
+<table align="center" class="doxtable">
+  <tr>
+    <td>
+        @image html step_87_rising_bubble.png
+    </td>
+  </tr>
+</table>
+
+The discrete interface $\Gamma$ is
+represented implicitly through a certain isosurface of the level-set function
+e.g. for the signed-distance function $\Gamma=\{\boldsymbol x \in \Omega~|~\phi(\boldsymbol{x})=0\}$.
+We would like to note that deal.II provides a set of analytical signed distance
+functions for simple geometries in the Functions::SignedDistance namespace.
+Those can be combined via Boolean operations to describe more complex
+geometries @cite burman2015cutfem. The temporal evolution of the level-set field
+is obtained by the transport equation
+@f[
+	\frac{\partial \, \phi}{\partial \, t} + \boldsymbol{u}\vert_\Gamma \cdot \nabla \phi = 0
+@f]
+with the transport velocity at the interface $\boldsymbol{u}\vert_\Gamma$,
+which might be approximated by the local fluid velocity $\boldsymbol{u}\vert_\Gamma\approx\boldsymbol{u}(\boldsymbol{x})$. To reobtain the
+signed-distance property of the level-set field throughout the numerical solution procedure, PDE-based or,
```

Comment:
```suggestion
signed-distance property of the level-set field throughout the numerical solution procedure, PDE-based or,
```
I find that part a bit convoluted to read. Is there a way to make that sentence clearer?

Reply:
I tried to reformulate it a bit to make it clearer.